importexport/musicxml/importmxmlpass2.cpp

```diff
@@ -2453,6 +2454,20 @@ static void preventNegativeTick(const Fraction& tick, Fraction& offset, MxmlLogg
             }
       }
 
+//---------------------------------------------------------
+//   isTempoOrphanCandidate
+//---------------------------------------------------------
+
+bool MusicXMLDelayedDirectionElement::isTempoOrphanCandidate() const
+      {
+      return _element->isStaffText()
+            && _placement == "above"
+            && isBold();
```

Comment:
looks much cleaner! Perhaps _placement shouldn't be a string, but I gather that's as per the MusicXML standard.